### ISSUE TITLE
Typo in `Style/TrivialAccessors`'s `AllowedMethods`

### ISSUE THREAD

**Issue by movermeyer** [rubocop-hq/rubocop#7530]
The name of the `Whitelist` parameter of `Style/TrivialAccessors` was recently changed to `AllowedMethods`.

However, there were two places where the name was accidentally changed to `AllowedMethod` instead (Note the missing trailing `s`).

--------

## Expected behavior

The naming should be consistent and not give any warnings.

## Actual behavior

Incorrect warning message.

```
Warning: Style/TrivialAccessors does not support AllowedMethods parameter.

Supported parameters are:

  - Enabled
  - ExactNameMatch
  - AllowPredicates
  - AllowDSLWriters
  - IgnoreClassMethods
  - AllowedMethod
```

## Steps to reproduce the problem

Run `rubocop -a` on any file. The above warning is printed to the console.

## RuboCop version

```
rubocop -V
0.77.0 (using Parser 2.6.5.0, running on ruby 2.6.5 x86_64-darwin19)
```

cc @bbatsov
